packages/documentation/src/templates/page.js

```diff
@@ -9,12 +9,8 @@ export default ({ location, pageContext }) => {
   const content = <div dangerouslySetInnerHTML={{ __html: html }} />;
 
   return (
-    <SidebarLayout location={location}>
-      <div>{content}</div>
-      <div>
-        <h5>Source:</h5>
-        <a href={sourceUrl}>{sourceUrl}</a>
-      </div>
+    <SidebarLayout location={location} pageContext={pageContext}>
```

Comment:
I acknowledge there's some duplication here. There might be a better way of combining some of these templates/layouts, but to keep this PR focused, I'll save that refactor for another day...a

Reply:
I'll happily accept a small amount of duplication over a suboptimal abstraction. 

Reference: 
- https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction
- https://kentcdodds.com/blog/aha-programming#aha-

Reply:
For sure, and the refactor could just be renaming all of the templates in a morre consistent and clear way.